src/h5web/visualizations/shared/TooltipMesh.tsx

```diff
@@ -1,19 +1,22 @@
 import React, { ReactElement, useCallback } from 'react';
 import { Dom, PointerEvent, useThree } from 'react-three-fiber';
 import { TooltipWithBounds, useTooltip } from '@vx/tooltip';
+import { Line } from '@vx/shape';
 
 import styles from './TooltipMesh.module.css';
 import { useAbscissaScale, useOrdinateScale } from './utils';
 
 type Coords = [number, number];
+type Direction = 'horizontal' | 'vertical';
 
 interface Props {
   formatIndex: (t: Coords) => string;
   formatValue: (t: Coords) => string | undefined;
+  guides?: Direction[];
```

Comment:
This array allows you to write `['horizontal', horizontal']` or `['vertical', 'horizontal']`, so it's a bit too unclear I think. It'd be easier to have a single value: 'horizontal' | 'vertical' | 'both' or an object with `{ horizontal: true, vertical: true }, perhaps.